src/pages/components/select.mdx

```diff
@@ -0,0 +1,101 @@
+---
+title: Select
+description: >
+  Select allows a user to pick one option from a list.
+  This helps simplify the UI when space is limited.
+package: '@zendeskgarden/react-dropdowns'
+propSheets:
+  - dropdowns/src/elements/Select/Select.tsx
+---
+
+<Usage>
+  <Use>
+    <ul>
+      <li>To make a selection from a list of options</li>
+    </ul>
+  </Use>
+  <Misuse>
+    <ul>
+      <li>
+        To filter a large list of options, consider an{' '}
+        <a href="/components/autocomplete">Autocomplete</a>
+      </li>
+      <li>
+        To make multiple selection from a list, use a{' '}
+        <a href="/components/multiselect">Multiselect</a>
```

Comment:
The Autocomplete page is being designed and development should begin shortly. The Multiselect component page is under development. I can change the href to `href="#TODO"` if preferable.

Reply:
I think this is the right option as it prevents future clean-up.